src/detect-engine-loader.h

```diff
@@ -44,9 +44,9 @@ typedef struct DetectLoaderTask_ {
 typedef struct DetectLoaderControl_ {
     int id;
     int result;     /* 0 for ok, error otherwise */
-    ThreadVars *tv; /* loader threads threadvars - for waking them up */
-    SCMutex m;
+    SCMutex m;      /**< mutex protectx result and task_list */
```

Comment:
minor typo: `protectx`
Q: Why did you do this reordering of struct items? To have the cacheline boundary fall outside of `task_list`?

Reply:
It's not a perf critical struct, so I reordered things to have the mutex and the members it protects grouped together

Reply:
ok. Thank you. This PR looks good to me. If you're fixing the typo in comment, I'll wait for the next PR to approve.